tests/testthat/helper.R

```diff
@@ -10,3 +10,53 @@ task_grp = TaskClassifST$new("ecuador-grp", as_data_backend(data),
   target = "slides",
   coordinate_names = c("x", "y"))
 task_grp$col_roles$group = "grp"
+
+# Create 6x6 point grid with 1m distance between points
+TEST_MAKE_SP = function() {
```

Comment:
Why all upper case?

Reply:
We are doing this in `mlr3tuning` for functions that create test objects (https://github.com/mlr-org/mlr3tuning/blob/master/tests/testthat/helper.R). However, we are not doing this mlr3.